src/App.js

```diff
@@ -1,72 +1,78 @@
-import React, { Component } from "react";
+import React, { Component } from 'react';
 
+import People from './People';
 class App extends Component {
+  // Classes must have constructors if initial variables are to be set automatically
   constructor() {
-    super()
+    // calls the constructor for the parent (Component) class
+    super();
+    // sets the initail state to include these items
     this.state = {
-      pokemon: [],
-      search: "",
-      selectedPokemon: null
-    }
+      nameInput: '',
+      ageInput: '',
+      people: [
+        { name: 'Alex', age: 21 },
+        { name: 'Ben', age: 18 },
+        { name: 'Abe', age: 31 }
+      ]
+    };
   }
 
-  async componentDidMount() {
-    const res = await fetch('https://pokeapi.co/api/v2/pokemon/')
-    const json = await res.json()
-    this.setState({pokemon: json.results})
-  }
+  addPerson = (event) => {
+    event.preventDefault();
+    const { people } = this.state;
+    // const people = this.state.people;
 
-  onSearchChange = event => {
-    this.setState({search: event.target.value})
-  }
+    const person = {
+      name: this.state.nameInput,
+      age: this.state.ageInput
+    };
+
+    people.push(person);
```

Comment:
Technically modifying in place, we can either fix this or discuss in class

Reply:
These have been updated